tests/testthat/test-write_dwc.R

```diff
@@ -22,11 +22,14 @@ test_that("write_dwc() returns expected files and messaging", {
 })
 
 test_that("write_dwc() returns the expected Darwin Core terms as columns", {
+  result <- readr::read_csv(
+    file.path(temp_dir, "dwc_occurrence.csv"),
+    n_max = 1,
+    show_col_types = FALSE
+  )
+
   expect_identical(
-    colnames(readr::read_csv(file.path(temp_dir, "dwc_occurrence.csv"),
-      n_max = 1,
-      show_col_types = FALSE
-    )),
+    colnames(result),
```

Comment:
`expect_named()` ?

Reply:
Is this a more appropriate test? If so, you can update.